Fix job application article

gchang12 opened the pull request:
### Description

Changed wording of 'en-US/docs/Learn_web_development/Getting_started/Soft_skills/Finding_a_job' for clarity's sake.

### Motivation

Readers won't have to suffer through mildly incorrect English.

### Additional details

Unable to erase commas after instances of 'i.e.' and 'e.g.'

gchang12 committed:
Fixed hyphenation.

gchang12 committed:
Fixed commas; added hyperlink to Medium.

gchang12 committed:
Fixed commas.

gchang12 committed:
Added hyperlink to Medium where appropriate.

gchang12 committed:
Added period to <li> content for consistency.

gchang12 committed:
Fixed wording so that technical writer description entails writing documentation that explains how a SPECIFIC technology works, rather than how technology in GENERAL works.

gchang12 committed:
Changed wording: guides that tell you how to write web-based CVs (vs) guides on the web that tell you how to write CVs

gchang12 committed:
Made content refer to GDocs files properly.

gchang12 committed:
Deleted erroneous comma (preceded dependent clause).

gchang12 committed:
Inserted 'e.g.' (for example) and 'i.e.' (in other words) where appropriate.

gchang12 committed:
Minor wording changes, plus a typo-fix.

gchang12 committed:
Capitalized 'Fibonacci'; made prompt more mathematically rigorous.

gchang12 committed:
Added more wording changes.

gchang12 committed:
Added in some wording to fill in a gap in sentence, so to speak

gchang12 committed:
Reworded more paragraphs for better flow.

gchang12 committed:
Added more wording changes for better flow.

gchang12 committed:
Reworded for clarity's sake.

gchang12 committed:
Replaced 'I' with 'you' in accordance with OG author's wording.

gchang12 committed:
Merge branch 'mdn:main' into fix-job-application-article

gchang12 committed:
Merge remote-tracking branch 'upstream/main' into fix-job-application-article

gchang12 committed:
Replaced dollar-signs with back-ticks.

gchang12 committed:
Merge branch 'fix-job-application-article' of https://github.com/gchang12/mdn-content into fix-job-application-article

gchang12 committed:
Removed commas after instances of 'i.e.' and 'e.g.'

gchang12 committed:
Removed commas after instances of 'e.g.' and 'i.e.'

gchang12 committed:
Removed commas after instances of 'e.g.' and 'i.e.'

gchang12 committed:
Removed commas after instances of 'e.g.' and 'i.e.'

gchang12 committed:
Removed commas after instances of 'e.g.' and 'i.e.'

gchang12 committed:
Removed commas after instances of 'e.g.' and 'i.e.'

chrisdavidmills reviewed (changes requested):
Hi there @gchang12, and thank you for the cleanup work! I have gone through all of your changes. Most of them make sense, but I've included a few further suggestions for you to look at.

chrisdavidmills committed:
Merge branch 'main' into fix-job-application-article

gchang12 committed:
Update files/en-us/learn_web_development/getting_started/soft_skills/finding_a_job/index.md

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

gchang12 committed:
Update files/en-us/learn_web_development/getting_started/soft_skills/finding_a_job/index.md

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

gchang12 committed:
Update files/en-us/learn_web_development/getting_started/soft_skills/finding_a_job/index.md

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

gchang12 committed:
Update files/en-us/learn_web_development/getting_started/soft_skills/finding_a_job/index.md

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

gchang12 committed:
Update files/en-us/learn_web_development/getting_started/soft_skills/finding_a_job/index.md

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

gchang12 committed:
Update files/en-us/learn_web_development/getting_started/soft_skills/finding_a_job/index.md

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

gchang12 committed:
Update files/en-us/learn_web_development/getting_started/soft_skills/finding_a_job/index.md

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

gchang12 committed:
Addded rewording to reduce ambiguity.

https://github.com/mdn/content/pull/41489/files/4b48283401a89718f3a1f40349007cf6817ec970..1fc686d9306141ecc3c9ab9bfdba841c112a95eb#diff-81bdf0a00e5a6bf1b9f17ad3528d98f32590d9f092413af1e897036cc71e33f1

gchang12 committed:
Update files/en-us/learn_web_development/getting_started/soft_skills/finding_a_job/index.md

Remove trailing space.

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

gchang12 committed:
Update index.md

Reverted this commit, which was sneakily put in without author's consent.

https://github.com/mdn/content/pull/41489/commits/9443fe5550e94593ddbb483ab6a2278462bf50b3

 Sorry.

chrisdavidmills reviewed (approved):
This looks great; let's get it merged. Thanks a lot for the improvements, @gchang12.

chrisdavidmills committed:
small final tweak

chrisdavidmills committed:
A few more small bits
